README.md

```diff
@@ -50,10 +52,47 @@ In addition to `live_view_native` you may want to include some additional librar
 {:live_view_native_live_form, "~> 0.3.0-rc.3"}
 ```
 
+The versions above may be subject to change, so make sure to check the library documentation for the latest version.
+* [LiveView Native Versions](https://hex.pm/packages/live_view_native/versions)
+* [LiveView Native Stylesheet Versions](https://hex.pm/packages/live_view_native_stylesheet/versions)
+* [LiveView Native SwiftUI Client Versions](https://hex.pm/packages/live_view_native_swiftui/versions)
+* [LiveView Native Live Form Versions](https://hex.pm/packages/live_view_native_live_form/versions)
+
 Then run:
 
 ```
 $ mix lvn.setup
 ```
 
 and follow the instructions on how to complete the setup process.
+
+## Main Branch
+
+To try out new features or get the latest bugfixes you can also use the `main` branch for LiveView Native dependencies.
+
+```elixir
```

Comment:
I'd rather not put this in the README as it may encourage more use of unreleased versions. That should be for development